tests/commands_tests/test_console_utils.py

```diff
@@ -1,199 +0,0 @@
-# Copyright (c) 2016 Shotgun Software Inc.
```

Comment:
_check_constraints doesn't exist anymore so everything has been moved to test_descriptors.py